prebuilt-config/bitops-tag.yaml

```diff
@@ -1,3 +1,3 @@
 tags:
-  bitops_base: plugins-base-RC
+  bitops_base: base
   plugins_base: plugins
```

Comment:
As a side note, the use case for this file is dangerous as the workflow here https://github.com/bitovi/bitops/blob/main/.github/workflows/bump-image-tag.yaml can update the values in that file anytime, breaking the default assumptions and changing the entire workflow results.

The values for the base tag should not be overridden and only the merges to `main` branches and versioned github release/tag should produce an artifact (Docker image) with predictable naming, without relying on additional yaml.

I propose removing it in the future iterations.

Reply:
@armab is there an issue/discussion for this?